data/ams0.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "Alessandro Vozza",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Platform Advocate @ Solo.io | Organizer at Kubernetes Community Days Amsterdam | Kubernetes | Service Mesh | Cloud-Native Open Source",
+  "tags": [
+    "DevRel",
+    "Cloud-Native Open Source",
+    "DevOps",
+    "Kubernetes",
+    "Service Mesh",
+    "Networking"
+  ],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/bongo" },
+    { "icon": "FaGithub", "url": "https://github.com/ams0" },
+    { "icon": "FaLinkedIn", "url": "https://www.linkedin.com/in/alessandrovozza/" }
+  ],
+  "avatar": "https://github.com/ams0.png",
```

Comment:
As of our latest update, we no longer need the `avatar` key. So please remove this from your file. 🛠

```suggestion
```